activation/activation.go

```diff
@@ -118,6 +118,12 @@ func WithPoetConfig(c PoetConfig) BuilderOption {
 	}
 }
 
+func WithLogger(l log.Logger) BuilderOption {
```

Comment:
The builder now receives the `log` instance twice: as a parameter and as a functional option. Can we just use one of the two? I'd prefer `WithLogger` but I'm also OK with not having the functional option and instead just passing it in as a parameter.

Reply:
Yeah, that's not great. I think its better to go with passing the param directly, since as my understanding of the option pattern is that it is reserved for optional parameters and the log is not optional.

Reply:
@fasmat & @dshulyak I removed the `WithLogger` option but that was before @dshulyak suggested using log.Nop, so If you think It's worth it I can switch back to using `WithLogger` with  log.Nop ?